torch/csrc/jit/python/pybind_utils.h

```diff
@@ -46,12 +46,21 @@
 namespace torch {
 namespace jit {
 
+py::object toPyObject(IValue ivalue);
+
 // The PythonFutureWrapper for ivalue::Future
-struct PythonFutureWrapper {
-  explicit PythonFutureWrapper(c10::intrusive_ptr<c10::ivalue::Future> fut)
-      : fut(std::move(fut)) {}
+struct VISIBILITY_HIDDEN PythonFutureWrapper {
+  using UnwrapFunc = std::function<void(py::object)>;
+
+  explicit PythonFutureWrapper(
+      c10::intrusive_ptr<c10::ivalue::Future> fut,
+      c10::optional<UnwrapFunc> unwrap_func = c10::nullopt)
+      : fut(std::move(fut)), unwrap_func(std::move(unwrap_func)) {}
```

Comment:
This problem exists prior to this PR. I wonder if we should use `fut_` and `unwrap_func_` to stay consistent with the `InferredType` impl below

Reply:
Not planning to change this. I noticed it's a JIT code base naming convention that they use Python like naming. `InferredType` is an outlier.